pkg/asset/machines/vsphere/machines.go

```diff
@@ -3,15 +3,15 @@ package vsphere
 
 import (
 	"fmt"
+	"regexp"
 
 	machineapi "github.com/openshift/api/machine/v1beta1"
+	"github.com/openshift/installer/pkg/types"
+	"github.com/openshift/installer/pkg/types/vsphere"
```

Comment:
nit: these imports should go in there own block, we try to group them:

<stdlib>

<external>

<internal>

We need to document this/enforce this